src/main/java/it/hospital/project/appusers/User.java

```diff
@@ -1,24 +1,42 @@
 package it.hospital.project.appusers;
 
 
+import org.hibernate.annotations.GenericGenerator;
+
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.Table;
 
-import lombok.Data;
-import lombok.NoArgsConstructor;
-
-
-
-@Data
-@NoArgsConstructor
 @Entity
 @Table(name = "appuser")
 public class User {
 
+    public User() {
+    }
+
+    // constructor for tests
```

Comment:
This comment is unnecessary

Reply:
The proper way of marking in-progress comments is like this:

// TODO: constructor for tests

Reply:
I meant, that this comment is unnecessary. Constructor is already done.